asciidoctorj-core/src/main/java/org/asciidoctor/Options.java

```diff
@@ -67,10 +67,26 @@ public void setTemplateEngine(String templateEngine) {
         this.options.put(TEMPLATE_ENGINE, templateEngine);
     }
 
+    /**
+     * Enable writing output to a file. The file includes header and footer by default.
+     *
+     * @param toFile The path to the output file. If the path is not absolute, it is
+     *               interpreted relative to what was set via {@link #setToDir()}
+     *               or {@link #setBaseDir()} respectively.
+     *
+     */
     public void setToFile(String toFile) {
         this.options.put(TO_FILE, toFile);
     }
 
+    /**
+     * Toogle writing output to a file.
+     *
+     * @param toFile If <code>true</true>, overwrite the input file with the output,
```

Comment:
It's also not true that it "overwrite(s) the input file with the output". It writes to a file adjacent to the input file which has the same root name and a file extension as specified by the converter.